openreview/conference/helpers.py

```diff
@@ -64,7 +64,8 @@ def get_conference_builder(client, request_form_id):
         'deadline': 'Submission Start: ' + submission_start_date_str + ' UTC-0, End: ' + submission_due_date_str + ' UTC-0',
         'date': conference_start_date_str,
         'website': note.content['Official Website URL'],
-        'location': note.content.get('Location')
+        'location': note.content.get('Location'),
+        'contact': note.content.get('contact_email')
```

Comment:
Should this be -> note.content.get('Contact Emails')

Reply:
Oh, I see the change in https://github.com/openreview/openreview-scripts/pull/1161.